src/main/java/gregtech/common/tileentities/machines/multi/MTEEnvironmentallyControlledChemicalFacility.java

```diff
@@ -0,0 +1,1076 @@
+                .addIcon(OVERLAY_FRONT_LARGE_CHEMICAL_REACTOR_ACTIVE)
+                .extFacing()
+                .build(),
+                TextureFactory.builder()
+                    .addIcon(OVERLAY_FRONT_LARGE_CHEMICAL_REACTOR_ACTIVE_GLOW)
+                    .extFacing()
+                    .glow()
+                    .build() };
+            return new ITexture[] { casingTexturePages[1][48], TextureFactory.builder()
+                .addIcon(OVERLAY_FRONT_LARGE_CHEMICAL_REACTOR)
+                .extFacing()
+                .build(),
+                TextureFactory.builder()
+                    .addIcon(OVERLAY_FRONT_LARGE_CHEMICAL_REACTOR_GLOW)
+                    .extFacing()
+                    .glow()
+                    .build() };
+        }
+        return new ITexture[] { casingTexturePages[1][48] };
+    }
+
+    @Override
+    protected MultiblockTooltipBuilder createTooltip() {
+        MultiblockTooltipBuilder tt = new MultiblockTooltipBuilder();
+        tt.addMachineType("Environmentally Controlled Chemical Facility, ECCF")
+            .addInfo(
+                EnumChatFormatting.GRAY + "Allows to produce some chemicals in a "
+                    + EnumChatFormatting.BLUE
+                    + "single step"
+                    + EnumChatFormatting.GRAY
+                    + ", but requires special conditions")
+            .addInfo(
+                EnumChatFormatting.GRAY + "Doesn't overclock, instead increases parallels by "
+                    + EnumChatFormatting.GOLD
+                    + "4 ^ Energy Tier")
+            .addSeparator()
+            .addInfo(
+                EnumChatFormatting.GRAY + "Conditions are shown in NEI and can be achieved by placing ECCF on another "
+                    + EnumChatFormatting.BLUE
+                    + "planet ")
+            .addInfo(
+                EnumChatFormatting.GRAY + "with required conditions or maintaining them using "
+                    + EnumChatFormatting.BLUE
+                    + "Modules")
+            .addSeparator()
+            .addInfo("" + EnumChatFormatting.WHITE + EnumChatFormatting.BOLD + "Module interaction")
+            .addInfo(
+                EnumChatFormatting.GRAY + "ECCF drains "
+                    + EnumChatFormatting.YELLOW
+                    + "all coolants"
+                    + EnumChatFormatting.GRAY
+                    + " from the input hatch in the temperature module")
+            .addInfo(
+                EnumChatFormatting.GRAY + "Every second, ECCF temperature drifts towards ambient, to: "
+                    + EnumChatFormatting.GOLD
+                    + "(current - ambient) * 0.98 + ambient")
+            .addInfo(
+                EnumChatFormatting.GRAY + "And applies temperature changes regardless of the module type: "
+                    + EnumChatFormatting.GOLD
+                    + "current + (fluid amount / 10000) * coolant value")
```

Comment:
Thought of a potentially smoother/clearer ordering
```suggestion
            .addInfo(
                EnumChatFormatting.GRAY + "Every second, ECCF temperature drifts towards ambient, to: "
                    + EnumChatFormatting.GOLD
                    + "(current - ambient) * 0.98 + ambient")
            .addInfo(
                EnumChatFormatting.GRAY + "It drains "
                    + EnumChatFormatting.YELLOW
                    + "all coolants"
                    + EnumChatFormatting.GRAY
                    + " from the input hatch in the temperature module")
            .addInfo(
                EnumChatFormatting.GRAY + "And changes temperature to: "
                    + EnumChatFormatting.GOLD
                    + "current + (fluid amount / 10000) * coolant value")
```